### PR TITLE
Enhance/ Responsive update for the ContactUs and LatestNews section #75

### PR DESCRIPTION
**Pull request by Charles-Xing**
### Descriptions

- Link to UI design (if possible):https://www.figma.com/file/q2ZHM8s0LcNxz3otGFLDzO/Website?node-id=0%3A1&t=U1p59QlVKt5XJoh5-0

### Provide screenshots or videos that the function is working.
ContactUs section-iPad mini
image
ContactUs section-iPhone SE
image

LatestNews section-iPad mini
image
image
LatestNews section-iPhone SE
image
image

### Provide screenshots of no warning, or error in the console.
image

- [X] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [X] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [X] Check PR title is aligned with your ticket.
- [X] Check your app will run without crashes.
- [X] Check if there are hard-coded values.
- [X] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [X] Check you are using `camelCase && PascalCase` correctly.
- [X] Remove unused comments.

### CODE REVIEW

File: components/general/InfoBlock/InfoBlock.module.css [hunk 1]
```diff
@@ -108,3 +107,15 @@
...
   pointer-events: none;
 }
+
+@media screen and (max-width: 1023px) {
+  .info_block__normal_content {
+    display: none;
+  }
+}
+
+/* @media screen and (max-width: 743px) {
```
Comment (HarryuCATGREY) [inline components/general/InfoBlock/InfoBlock.module.css:117]: Why you leave this comment?

File: jest.setup.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import "@testing-library/jest-dom/extend-expect";
 Object.defineProperty(window, "matchMedia", {
-  writable: true,
+  writable: false,
```
Comment (HarryuCATGREY) [inline jest.setup.js:3]: Why you change this parameter?

File: components/general/InfoBlock/InfoBlock.module.css (hunk 1 above)
Comment (Charles-Xing) [reply components/general/InfoBlock/InfoBlock.module.css:117]: I will delete it, thx

Comment (Charles-Xing) [reply components/general/InfoBlock/InfoBlock.module.css]: An error has been corrected.

**Review by yunshen513 — APPROVED** [review]
LGTM

**Review by HarryuCATGREY — APPROVED** [review]
LGTM

File: components/general/HowWeWork.tsx [hunk 3]
```diff
@@ -1,6 +1,10 @@
 import InfoBlock from "./InfoBlock/InfoBlock";
 
-const HowWeWork = () => {
+type Props = {
+  isPhoneSize: boolean;
+};
+
+const HowWeWork = ({ isPhoneSize }: Props) => {
```
Comment (Cameron933) [inline components/general/HowWeWork.tsx:7]: Your task should be responsive update for ContactUs and LastestNews, so why you change this component?  I do not understand

Comment (Cameron933) [reply components/general/HowWeWork.tsx:7]: image

File: components/general/Subscription.tsx [hunk 4]
```diff
@@ -1,13 +1,18 @@
 import InfoBlock from "./InfoBlock/InfoBlock";
 
-const Subscription = () => {
```
Comment (Cameron933) [inline components/general/Subscription.tsx:3]: Same, do not even belong to your task.

**Review by Cameron933 — COMMENTED** [review]
image

File: components/general/InfoBlock/InfoBlock.tsx [hunk 5]
```diff
@@ -10,6 +10,7 @@ type InfoProps = {
   blockType: InfoBlockType;
   content: string;
   buttonContent?: string;
+  isPhoneSize: boolean;
```
Comment (Cameron933) [inline components/general/InfoBlock/InfoBlock.tsx:13]: image
Make it optional, so you need not change all those components which based on this one.

**Pull request merged**